common/scaffolding/typographie.html

```diff
@@ -0,0 +1,45 @@
+---
+{
+	"title": "Typographie",
+	"language": "fr",
+	"altLangPage": "typography.html",
+	"breadcrumbs": [
+		{ "title": "GCWeb home", "link": "https://wet-boew.github.io/GCWeb/index-en.html" }
+	],
+	"dateModified": "2024-03-06"
+}
+---
+
+<p>Le contenu suivant utilise des en-têtes sémantiques HTML pour démontrer leur rendu et leur style. Un paragraphe sur le rôle de l'aria leur a été ajouté pour minimiser l'impact de la technologie d'assistance.</p>
+
+<h2>Titres</h2>
+
+<h1 role="paragraph">Il s'agit d'un élément de titre h1</h1>
+<h2 role="paragraph">Il s'agit d'un élément de titre h2</h2>
+<h3 role="paragraph">Il s'agit d'un élément de titre h3</h3>
+<h4 role="paragraph">Il s'agit d'un élément de titre h4</h4>
+<h5 role="paragraph">Il s'agit d'un élément de titre h5</h5>
+<h6 role="paragraph">Il s'agit d'un élément de titre h6</h6>
+<pre><code>
+&lt;h1&gt;Il s'agit d'un élément de titre h1&lt;/h1&gt;
+&lt;h2&gt;Il s'agit d'un élément de titre h2&lt;/h2&gt;
+&lt;h3&gt;Il s'agit d'un élément de titre h3&lt;/h3&gt;
+&lt;h4&gt;Il s'agit d'un élément de titre h4&lt;/h4&gt;
+&lt;h5&gt;Il s'agit d'un élément de titre h5&lt;/h5&gt;
+&lt;h6&gt;Il s'agit d'un élément de titre h6&lt;/h6&gt;
+</code></pre>
+
+<h2>Paragraphe</h2>
+
+<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
+<pre><code>
+&lt;p&gt;Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.&lt;/p&gt;
+</code></pre>
+
+<h2>Largeur de contenu limitée</h2>
+
+<p>Pour un exemple de contenu de largeur limitée, visitez le lien suivant: <a href="https://wet-boew.github.io/GCWeb/templates/content-limit-en.html">Page de contenu - Largeur de contenu limitée</a></p>
+
+<h2>Sémantique au niveau du texte</h2>
+
+<p>Pour un exemple de sémantique au niveau du texte qui présente les liens et regroupe les éléments associés, visitez le lien suivant: <a href="https://wet-boew.github.io/GCWeb/common/text-level-semantics/text-level-semantics.html">Sémantique au niveau du texte</a></p>
```

Comment:
Fix the broken link and use relative link instead of absolute URLs.

Reply:
Fixed and addressed in commit "Fixed links and removed breadcrumbs".